shark-android/src/main/java/shark/AndroidObjectInspectors.kt

```diff
@@ -361,6 +361,19 @@ enum class AndroidObjectInspectors : ObjectInspector {
         } else {
           notLeakingReasons += mQuitting describedWithValue "false"
         }
+
+        val queueHead = instance["android.os.MessageQueue", "mMessages"]!!.valueAsInstance
```

Comment:
Here and below - you explicitly want to crash if these fields are not found, correct? Otherwise it can probably be `?.`

Reply:
Yep, MessageQueue should always have mMessages and if not someone should report the error and we can fix it.